hack/rpm-deps.sh

```diff
@@ -102,7 +102,12 @@ bazel run \
     //:bazeldnf -- rpmtree --public --name libguestfs-tools \
     $basesystem \
     libguestfs \
-    libguestfs-tools
+    libguestfs-tools \
+    --force-ignore-with-dependencies '^(kernel-|linux-firmware)' \
+    --force-ignore-with-dependencies '^(python[3]{0,1}-|perl[3]{0,1}-)' \
+    --force-ignore-with-dependencies '^(mesa-|libwayland-|selinux-policy|mozjs60)' \
```

Comment:
Isn't dropping SELinux-related packages a bit too aggressive? Are you sure that's not going to cause any issues?

Reply:
I am looking forward to the e2e test results with libguestfs :)

Reply:
@andreabolognani I think it should be fine to drop selinux packages. Selinux runs outside the container not inside.